### PR TITLE
Tests for UI

### PR DESCRIPTION
**Pull request by inpego**

### CODE REVIEW

File: apps/web/templates/application.html.slim [hunk 1]
```diff
@@ -20,7 +20,7 @@ html
 
     = render partial: 'shared/components'
 
-    - if Hanami.env?(:development)
+    - if Hanami.env?(:development) || Hanami.env?(:test)
```
Comment (davydovanton) [inline apps/web/templates/application.html.slim]: `unless Hanami.env?(:production)`?

Comment (inpego) [inline apps/web/templates/application.html.slim]: Yep.

**Comment by inpego** [comment]
Point 4 added.

**Comment by inpego** [comment]
Points 2 and 3 added, ready to merge.

**Comment by inpego** [comment]
Fixed Travis-CI error.

**Comment by davydovanton** [comment]
great work, thanks! 👍

**Pull request merged**